File: sdks/redlite-java/src/main/java/com/redlite/RedliteException.java
package com.redlite;

/**
 * Exception thrown by Redlite operations.
 */
public class RedliteException extends RuntimeException {

    public RedliteException(String message) {
        super(message);
    }

    public RedliteException(String message, Throwable cause) {
        super(message, cause);
    }
}

File: sdks/redlite-dotnet/src/RedliteException.cs
namespace Redlite;

/// <summary>
/// Exception thrown by Redlite operations.
/// </summary>
public class RedliteException : Exception
{
    public RedliteException()
    {
    }

    public RedliteException(string message) : base(message)
    {
    }

    public RedliteException(string message, Exception innerException) : base(message, innerException)
    {
    }
}
